docs/people-and-groups/user-provisioning.md

```diff
@@ -0,0 +1,52 @@
+---
+title: User provisioning with SCIM
+---
+
+# User provisioning with SCIM
+
+{% include plans-blockquote.html feature="User provisioning with SCIM" %}
+
+Metabase supports user provisioning via the System for Cross-domain Identity Management (SCIM) protocol. In addition to Single Sign-on (SSO), you can set up user provisioning in Metabase with SCIM to:
+
+- **Decouple authentication from provisioning**. Even though anyone could authenticate with SSO, you may only want some groups to be able to create an account in Metabase.
+- **Support deprovisioning user accounts**. If you deactivate someone from your SSO, SCIM can let Metabase know to deactivate their Metabase account as well.
+
+> For now, Metabase supports SCIM for [Okta](https://developer.okta.com/docs/concepts/scim/) and [Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/architecture/sync-scim).
+
+## Setting up user provisioning
+
+To set up user provisioning. Click on the settings **Gear** icon in the upper right and navigate to **Admin settings** > **Settings**> **Authentication**.
+
+Click on the **User provisioning** tab.
+
+## User provisioning via SCIM
+
+To set up user provisioning with SCIM, hit the toggle to enable it. Metabase will tell you the SCIM endpoint URL and SCIM token to share with your identity provider
```

Comment:
Would it be worth explicitly having an Okta-specific setup guide? (and maybe Entra ID?) We have that sort of thing for SAML already.

e.g. here's Github's SCIM docs which tell you exactly what to do in Okta: https://docs.github.com/en/enterprise-server@3.14/admin/managing-iam/provisioning-user-accounts-with-scim/configuring-scim-provisioning-with-okta#2-configure-scim

Reply:
For now, I've updated the PR with links to the Okta and Entra  user provisioning setup docs. In general, I prefer not to recreate third-party docs (to avoid overhead, content drift, and inaccuracies), but we can add provider-specific guides later on if those third-party docs aren't clear.